src/material-experimental/mdc-progress-bar/harness/progress-bar-harness.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness} from '@angular/cdk-experimental/testing';
+
+/**
+ * Harness for interacting with a standard mat-progress-bar in tests.
+ * @dynamic
+ */
+export class MatProgressBarHarness extends ComponentHarness {
+  static hostSelector = 'mat-progress-bar';
+
+  /** Gets a promise for the progress bar's value. */
+  async getValue(): Promise<number|null> {
+    const host = await this.host();
+    const ariaValue = await host.getAttribute('aria-valuenow');
+
+    if (ariaValue) {
+      return Number(ariaValue);
```

Comment:
You can use `coerceNumberProperty` here to coerce the value.

Reply:
Good call - I added this here and in my PR for the progress spinner